apps/festival/festival/src/app/event/login/login.component.ts

```diff
@@ -19,6 +19,7 @@ export class EventLoginComponent implements OnInit, OnDestroy {
   });
   public buttonText = 'Log in';
   public signinIn = false;
+  public hidePassword: boolean = true;
```

Comment:
```suggestion
  public hidePassword = true;
```
Typescript is smart enough to recognise its type already